Default (Windows).sublime-keymap

```diff
@@ -0,0 +1,4 @@
+[
+    { "keys": ["ctrl+shift+f11"], "command": "rsync_ssh_sync_specific_remote" },
```

Comment:
I'm surprised that linux equivalent uses 'super' instead of 'ctrl'. I think it might not be working properly.